### DISCUSSION TITLE
can illa be integrated with open id connect for authentication?

### DISCUSSION THREAD

**Discussion by wuyh03** [illacloud/illa-builder#818] [Ideas]
like keycloak  etc，  so that we can use SSO .

**Comment by wenzhuo2022** [comment]
Thanks for your feedback. We are planning to integrate SSO providers but it may take some time. We will post a new feature announcement after it is available.

**Reply by wuyh03** [reply]
great, thanks!

**Reply by anime-shed** [reply]
Will be glad if this feature is available now?

**Reply by wenzhuo2022** [reply]
Not yet. We will support this feature by the end of this year.

**Reply by francoisauclair911** [reply]
Any updates? Thank you

**Reply by owenlongbo** [reply]
You can buy our enterprise version to support it. You can use Discord to connect us team if you want. Thx~

**Reply by francoisauclair911** [reply]
thanks so this feature won't be available as part of the free open source package?

**Reply by owenlongbo** [reply]
Yes.
